doc/source/getting-started/construct_backends.rst

```diff
@@ -0,0 +1,41 @@
+Setup a quantum machine learning backend
+========================================
+
+Among other features, Qiboml is a backend provider for Qibo. In particular, it can
+be used to construct differentiable backends to execute quantum machine learning
+algorithms. Currently, we support Tensorflow automatic differentiation, while
+Pytorch and Jax backends will be implemented in the next future.
```

Comment:
I think that now we support Pytorch as well, right?

Reply:
After the migration, yes :)

Reply:
Ok